app/views/projects/utils/inputSettings.scala.html

```diff
@@ -83,3 +84,18 @@ <h4>@messages("project.settings.license") <i>(@messages("general.optional"))</i>
     </div>
     <div class="clearfix"></div>
 </div>
+
+<div class="setting">
+    <div class="setting-description">
+        <h4>@messages("project.settings.postToForum")</h4>
+        <p>@messages("project.settings.postToForum.info")</p>
+    </div>
+    <div class="setting-content">
+        <label>
+            <input @if(postReleasesToForum) { checked } value="true" form="@form" type="checkbox"
+                    class="form-control" id="forums" name="post-to-forums">
```

Comment:
Due to the `form-control` the text is on a new line and it looks strange:
<img width="890" alt="schermafbeelding 2018-03-03 om 18 02 58" src="https://user-images.githubusercontent.com/3961526/36937013-3c960e86-1f0d-11e8-9c51-9fdfc14d1595.png">
I think its better like this
<img width="924" alt="schermafbeelding 2018-03-03 om 18 04 19" src="https://user-images.githubusercontent.com/3961526/36937017-4fbe66a2-1f0d-11e8-9993-08244bcc9f9d.png">